cis/system/ksp-audit-cis-mysql-2-1-4.yaml

```diff
@@ -0,0 +1,22 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-cis-mysql-2-1-4
+  namespace: default
+spec:
+  tags: ["MySQL", "CIS"]
```

Comment:
can we have the cis rule number also here? @yasin-cs-ko-ak

Reply:
@vishnusomank @salman-accuknox It is added. Thank you for reviewing.